client/asset/dcr/dcr_test.go

```
@@ -162,9 +162,8 @@ func tNewWalletMonitorBlocks(monitorBlocks bool) (*ExchangeWallet, *tRPCClient,
 	wallet.ctx = walletCtx
 
 	// Initialize the best block.
-	wallet.tipMtx.Lock()
-	wallet.currentTip, _ = wallet.getBestBlock(walletCtx)
```

Is this ignoring an error return?